sormas-api/src/main/resources/enum.properties

```diff
@@ -504,6 +504,16 @@ EventStatus.Short.SIGNAL = Signal
 EventStatus.Short.SCREENING = Screening
 EventStatus.Short.CLUSTER = Cluster
 
+# EventManagementStatus
+EventManagementStatus.PENDING = Pending
+EventManagementStatus.ONGOING = Ongoing
+EventManagementStatus.DONE = Done
+EventManagementStatus.CLOSED = Closed
+EventManagementStatus.Short.PENDING = Pending
```

Comment:
Short captions aren't necessary in this case because the original strings are already pretty short (and they don't differ at all)